### ISSUE TITLE
Link to Youtube video tutoral is bad

### ISSUE THREAD

**Issue by seant325** [Darakah/obsidian-timelines#70]
The link given in the README to the Youtube video is dead.

https://www.youtube.com/watch?v=_gtpZDXWcrM

**Comment by JustinaYuu** [comment]
yes!!!!! It's quite hard to learn how to use this plugin without video!!!

**Comment by seanlowe** [comment]
Hey guys, the whole plugin is kind of dead. I'm actively maintaining a fork [here](https://github.com/seanlowe/obsidian-timelines). I've written an easy-to-read [docs site](https://seanlowe.github.io/obsidian-timelines/docs/) to go along with it.

If there's enough demand, I could try to create an equivalent video for the fork.
